packages/dso-toolkit/src/components/accordion/accordion.models.ts

```diff
@@ -16,7 +17,8 @@ export interface Accordion<TemplateFnReturnType> {
 
 export interface AccordionSection<TemplateFnReturnType> {
   open?: boolean;
-  handleTitle: string;
+  handleTitle: RenvooiValue;
+  handleTitleText?: string;
```

Comment:
Wat is dit? Waarom zit dit in het model? Het is niet een property van het web component `Accordion` of `Accordion Section`.

Ik begrijp het niet.